common/dtd/ldml.dtd

```diff
@@ -1441,6 +1441,8 @@ CLDR data files are interpreted according to the LDML specification (http://unic
     <!--@DEPRECATED:true, false-->
 <!ATTLIST pattern references CDATA #IMPLIED >
     <!--@METADATA-->
+<!ATTLIST pattern numberSystem CDATA #IMPLIED >
+    <!--@MATCH:bcp47/nu-->
```

Comment:
I guess this does have to be #IMPLIED and not #REQUIRED because this is an existing element for which we did not formerly require a numberSystem attribute. Sorry for my mistaken earlier comment on this.